src/bin/taker.rs

```diff
@@ -168,6 +176,39 @@ fn main() {
             let address = taker.get_wallet_mut().get_next_external_address().unwrap();
             println!("{:?}", address);
         }
+        Commands::SendToAddress { address, amount } => {
+            // Assuming fee=fee_rate
+            // TODO: find a way to derive fee.
+            let fee = Amount::from_sat(args.fee_rate);
+            let send_amount = SendAmount::Amount(Amount::from_sat(amount));
+            let destination =
+                Destination::Address(Address::from_str(&address).unwrap().assume_checked());
+
+            let coins_to_spend = taker.get_wallet().list_all_utxo_spend_info(None).unwrap();
+            // create a signed tx
+            let tx = taker
+                .get_wallet_mut()
+                .spend_from_wallet(fee, send_amount, destination, &coins_to_spend)
+                .unwrap();
+
+            // derive RPC client
+            let rpc = Client::try_from(&rpc_config).unwrap();
+
+            // send the tx to network
+            rpc.send_raw_transaction(&tx).unwrap();
+
+            // confirm balance( Generate blocks)
+            let mining_address = rpc
+                .get_new_address(None, None)
+                .unwrap()
```

Comment:
Move this from send to address function to test framework. In real world, the taker-cli app should not mine blocks. 

Checkout in current test framework how we use the `BitcoinD` struct to generate new blocks and mining address.